src/main/java/thePackmaster/powers/warriorpack/TurtlePower.java

```diff
@@ -0,0 +1,55 @@
+package thePackmaster.powers.warriorpack;
+
+import com.evacipated.cardcrawl.mod.stslib.damagemods.AbstractDamageModifier;
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.DamageModApplyingPower;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDiscardAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cardmodifiers.warriorpack.FeralDamage;
+import thePackmaster.cards.warlockpack.Imp;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import java.util.Collections;
+import java.util.List;
+
+public class TurtlePower extends AbstractPackmasterPower implements DamageModApplyingPower {
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID(TurtlePower.class.getSimpleName());
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public TurtlePower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF,false, owner, amount);
+    }
+
+    @Override
+    public void updateDescription() {
+        this.description = (this.amount == 1 ? DESCRIPTIONS[0] : DESCRIPTIONS[1] + amount + DESCRIPTIONS[2]);
+    }
+
+    @Override
+    public boolean shouldPushMods(DamageInfo damageInfo, Object o, List<AbstractDamageModifier> list) {
+        return o instanceof AbstractCard && ((AbstractCard) o).type == AbstractCard.CardType.ATTACK;
+    }
+
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (card.type == AbstractCard.CardType.ATTACK && card.baseDamage > 0) {
+            --this.amount;
+            this.flash();
+            if (amount <= 0)
+                this.addToBot(new RemoveSpecificPowerAction(this.owner, this.owner, this));
+            this.updateDescription();
+        }
+    }
+
+    @Override
+    public List<AbstractDamageModifier> modsToPush(DamageInfo damageInfo, Object o, List<AbstractDamageModifier> list) {
+        return Collections.singletonList(new FeralDamage());
```

Comment:
Does this remove other modifiers that were in the given list?

Reply:
Works with in combination with my Front damage so no it doesn't ig ?